test/wpool_process_SUITE.erl

```diff
@@ -175,24 +175,19 @@ continue(_Config) ->
 
 -spec format_status(config()) -> {comment, []}.
 format_status(_Config) ->
-    %% echo_server implements format_status/2
+    %% echo_server implements format_status/1
     {ok, Pid} = wpool_process:start_link(?MODULE, echo_server, {ok, state}, []),
-    %% therefore it returns {formatted_state, State} as its status
-    {status, Pid, {module, gen_server}, SItems} = sys:get_status(Pid),
-    [state] = [S || SItemList = [_ | _] <- SItems, {formatted_state, S} <- SItemList],
```

Comment:
Removed for simplicity, since I'm not actually sure we're "testing" anything here, and, otoh, I'd have to replicate further code here (as the comment points out) in both pre-25 and 25+.